contracts/base-helper.ts

```diff
@@ -0,0 +1,120 @@
+import axios from  "axios";
+import fs from "fs";
+import { SigningCosmWasmClient } from "@cosmjs/cosmwasm-stargate";
+import { GasPrice, calculateFee, StdFee } from "@cosmjs/stargate";
+import { DirectSecp256k1HdWallet, makeCosmoshubPath } from "@cosmjs/proto-signing";
+import { Slip10RawIndex } from "@cosmjs/crypto";
+import path from "path";
+
+/*
+ * This base-helper is here to use with different helper files in contracts
+ *
+ * Usage: npx @cosmjs/cli@^0.26 --init https://raw.githubusercontent.com/CosmWasm/cw-plus/master/contracts/base-helpers.ts
+ *
+ * Create a client:
+ *   const [addr, client] = await useOptions(pebblenetOptions).setup('password');
+ *
+ * Get the mnemonic:
+ *   await useOptions(pebblenetOptions).recoverMnemonic(password);
+ *
+ * If you want to use this code inside an app, you will need several imports from https://github.com/CosmWasm/cosmjs
+*/
+
+interface Options {
+  readonly httpUrl: string
+  readonly networkId: string
+  readonly feeToken: string
+  readonly bech32prefix: string
+  readonly hdPath: readonly Slip10RawIndex[]
```

Comment:
You can use the alias

```suggestion
  readonly hdPath: HdPath
```

if the existing type looks too scary